postgis_eklentisi/postgis_eklentisi.md

```diff
@@ -0,0 +1,615 @@
+# ETKİN MEKANSAL VERİ YÖNETİM ARACI: POSTGIS
```

Comment:
Elinize sağlık açıklayıcı, uzunlkuğu ve bilgi düzeyi bence tam kararında çok başarılı bir yazı olmuş.

Reply:
İbrahim Bey, öncelikle katkılarınız ve değerlendirmeleriniz için teşekkür ederim. Postgis öğrenme serüvenime udemy platformundaki kursunuz ile başlamıştım. Sizden bu güzel yorumları almak manidar oldu. Teşekkürler.